app/controllers/OtherReasonController.scala

```diff
@@ -362,11 +359,11 @@ class OtherReasonController @Inject()(whenDidBecomeUnablePage: WhenDidBecomeUnab
         BadRequest(uploadEvidenceQuestionPage(formWithErrors, radioOptionsToRender, postAction))
       },
       isUploadEvidenceQuestion => {
-        logger.debug(
-          s"[OtherReasonController][onSubmitForUploadEvidenceQuestion] - Adding '$isUploadEvidenceQuestion' to session under key: ${SessionKeys.isUploadEvidence}")
-        Redirect(navigation.nextPage(UploadEvidenceQuestionPage, mode, Some(isUploadEvidenceQuestion)))
-          .addingToSession((SessionKeys.isUploadEvidence, isUploadEvidenceQuestion))
-      }
+            logger.debug(
+              s"[OtherReasonController][onSubmitForUploadEvidenceQuestion] - Adding '$isUploadEvidenceQuestion' to session under key: ${SessionKeys.isUploadEvidence}")
+            Redirect(navigation.nextPage(UploadEvidenceQuestionPage, mode, Some(isUploadEvidenceQuestion)))
+              .addingToSession((SessionKeys.isUploadEvidence, isUploadEvidenceQuestion))
+          }
```

Comment:
Could you remove the excess indentation?

Reply:
Yes 😃 , done